src/plugins/data/common/constants.ts

```diff
@@ -18,3 +18,5 @@
  */
 
 export const DEFAULT_QUERY_LANGUAGE = 'kuery';
+
+export const INDEX_PATTERN_MANAGEMENT_SECTION_PATH = '#/management/kibana/indexPatterns';
```

Comment:
I think it would make more sense for this to live in the Index Pattern Management plugin

Reply:
Honestly I don't think this even needs to be shared from the data plugin. Or at least, it doesn't make sense for the data plugin to "own" this in its contract... I'd probably vote for duplicating the string in each plugin for the time being if you are wary of having data depend on the index pattern management UI (which I understand).

Reply:
Unrelated question, is our convention for URLs to be camelCase? I can't remember but wanted to confirm.